libcxx/include/__expected/expected.h

```diff
@@ -282,10 +284,27 @@ class expected {
     }
   }
 
+  static constexpr bool __is_trivially_move_assignable =
+      is_trivially_move_constructible_v<_Tp> &&
+      is_trivially_move_assignable_v<_Tp> &&
+      is_trivially_move_constructible_v<_Err> &&
+      is_trivially_move_assignable_v<_Err>;
+
+  static constexpr bool __is_trivially_copy_assignable =
```

Comment:
This is incorrect. The types’ assignment operators were not used at all. 

If you assign an expected with value to an expected with error, what’s in the spec is to destroy + construct. 

so what you really need to check is the destructor and copy/move constructor.

Reply:
The conditions should be consistent with those of `variant`: [[variant.assign]/5](https://eel.is/c++draft/variant.assign#5), [[variant.assign]/10](https://eel.is/c++draft/variant.assign#10).